namer/k8s/src/main/scala/io/buoyant/namer/k8s/K8sExternalInitializer.scala

```diff
@@ -0,0 +1,54 @@
+package io.buoyant.namer.k8s
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle._
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.{ClientConfig, ServiceNamer}
+import io.buoyant.k8s.v1.Api
+import io.buoyant.namer.{NamerConfig, NamerInitializer}
+
+/**
+ * Supports namer configurations in the form:
+ *
+ * <pre>
+ * namers:
+ * - kind: io.l5d.k8s-external
+ *   exerimental: true
+ *   host: localhost
+ *   port: 8001
+ * </pre>
+ */
+class K8sExternalInitializer extends NamerInitializer {
+  val configClass = classOf[K8sExternalConfig]
+  override def configId = "io.l5d.k8s-external"
```

Comment:
this is the first/only dashed name in our kinds...  Perhaps io.l5d.k8s.external?